freeswitch needs host distutils

Maintainer: @micmac1 
Environment: fediora 35

Sounds like pythion needs to be checked in STAGING_DIR/hostpkg instead of the host OS.

```
checking for a BSD-compatible install... /home/mangix/devstuff/openwrt/staging_dir/host/bin/install -c
checking whether build environment is sane... yes
checking for mips64el-openwrt-linux-strip... mips64el-openwrt-linux-musl-strip
checking for a thread-safe mkdir -p... /usr/bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking whether make supports nested variables... (cached) yes
checking build system type... x86_64-pc-linux-gnu
checking host system type... mips64el-openwrt-linux-gnu
checking for style of include used by make... GNU
checking for mips64el-openwrt-linux-gcc... ccache_cc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... yes
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether ccache_cc accepts -g... yes
checking for ccache_cc option to accept ISO C89... none needed
checking whether ccache_cc understands -c and -o together... yes
checking dependency style of ccache_cc... none
checking for mips64el-openwrt-linux-gcc... (cached) ccache_cc
checking whether we are using the GNU C compiler... (cached) yes
checking whether ccache_cc accepts -g... (cached) yes
checking for ccache_cc option to accept ISO C89... (cached) none needed
checking whether ccache_cc understands -c and -o together... (cached) yes
checking dependency style of ccache_cc... (cached) none
checking whether we are using the GNU C++ compiler... yes
checking whether ccache_cxx accepts -g... yes
checking dependency style of ccache_cxx... none
checking whether the C++ compiler works... yes
checking for gawk... (cached) gawk
checking whether make sets $(MAKE)... (cached) yes
checking how to print strings... printf
checking for a sed that does not truncate output... /home/mangix/devstuff/openwrt/staging_dir/host/bin/sed
checking for grep that handles long lines and -e... /home/mangix/devstuff/openwrt/staging_dir/host/bin/grep
checking for egrep... /home/mangix/devstuff/openwrt/staging_dir/host/bin/grep -E
checking for fgrep... /home/mangix/devstuff/openwrt/staging_dir/host/bin/grep -F
checking for ld used by ccache_cc... mips64el-openwrt-linux-musl-ld
checking if the linker (mips64el-openwrt-linux-musl-ld) is GNU ld... yes
checking for BSD- or MS-compatible name lister (nm)... mips64el-openwrt-linux-musl-gcc-nm
checking the name lister (mips64el-openwrt-linux-musl-gcc-nm) interface... BSD nm
checking whether ln -s works... yes
checking the maximum length of command line arguments... 1572864
checking whether the shell understands some XSI constructs... yes
checking whether the shell understands "+="... yes
checking how to convert x86_64-pc-linux-gnu file names to mips64el-openwrt-linux-gnu format... func_convert_file_noop
checking how to convert x86_64-pc-linux-gnu file names to toolchain format... func_convert_file_noop
checking for mips64el-openwrt-linux-musl-ld option to reload object files... -r
checking for mips64el-openwrt-linux-objdump... mips64el-openwrt-linux-musl-objdump
checking how to recognize dependent libraries... pass_all
checking for mips64el-openwrt-linux-dlltool... no
checking for dlltool... no
checking how to associate runtime and link libraries... printf %s\n
checking for mips64el-openwrt-linux-ar... mips64el-openwrt-linux-musl-gcc-ar
checking for archiver @FILE support... @
checking for mips64el-openwrt-linux-strip... (cached) mips64el-openwrt-linux-musl-strip
checking for mips64el-openwrt-linux-ranlib... mips64el-openwrt-linux-musl-gcc-ranlib
checking command to parse mips64el-openwrt-linux-musl-gcc-nm output from ccache_cc object... ok
checking for sysroot... no
checking for mips64el-openwrt-linux-mt... no
checking for mt... no
checking if : is a manifest tool... no
checking how to run the C preprocessor... ccache_cc -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking for dlfcn.h... yes
checking for objdir... .libs
checking if ccache_cc supports -fno-rtti -fno-exceptions... no
checking for ccache_cc option to produce PIC... -fPIC -DPIC
checking if ccache_cc PIC flag -fPIC -DPIC works... yes
checking if ccache_cc static flag -static works... yes
checking if ccache_cc supports -c -o file.o... yes
checking if ccache_cc supports -c -o file.o... (cached) yes
checking whether the ccache_cc linker (mips64el-openwrt-linux-musl-ld) supports shared libraries... yes
checking whether -lc should be explicitly linked in... no
checking dynamic linker characteristics... GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... no
checking how to run the C++ preprocessor... ccache_cxx -E
checking for ld used by ccache_cxx... mips64el-openwrt-linux-musl-ld
checking if the linker (mips64el-openwrt-linux-musl-ld) is GNU ld... yes
checking whether the ccache_cxx linker (mips64el-openwrt-linux-musl-ld) supports shared libraries... yes
checking for ccache_cxx option to produce PIC... -fPIC -DPIC
checking if ccache_cxx PIC flag -fPIC -DPIC works... yes
checking if ccache_cxx static flag -static works... yes
checking if ccache_cxx supports -c -o file.o... yes
checking if ccache_cxx supports -c -o file.o... (cached) yes
checking whether the ccache_cxx linker (mips64el-openwrt-linux-musl-ld) supports shared libraries... yes
checking dynamic linker characteristics... (cached) GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking for C compiler vendor... gnu
checking libtool major version... 2
using libtool library extension... la
checking whether compiler supports -Wno-unused-result... yes
yes
checking whether compiler supports -Wno-misleading-indentation... yes
yes
checking whether compiler supports -Wno-error=format-truncation... yes
yes
  adding "-fPIC" to SWITCH_AM_CFLAGS
  adding "-ffast-math" to SWITCH_AM_CFLAGS
  adding "-fPIC" to SWITCH_AM_CXXFLAGS
  adding "-ffast-math" to SWITCH_AM_CXXFLAGS
  adding "-Werror" to SWITCH_AM_CFLAGS
  adding "-Wno-unused-result" to SWITCH_AM_CFLAGS
  adding "-Wno-misleading-indentation" to SWITCH_AM_CFLAGS
  adding "-Wno-error=format-truncation" to SWITCH_AM_CFLAGS
checking whether the compiler supports -fvisibility=hidden... yes
  adding "-fvisibility=hidden" to SWITCH_AM_CFLAGS
  adding "-DSWITCH_API_VISIBILITY=1" to SWITCH_AM_CFLAGS
  adding "-DCJSON_API_VISIBILITY=1" to SWITCH_AM_CFLAGS
  adding "-DHAVE_VISIBILITY=1" to SWITCH_AM_CFLAGS
  adding "-fvisibility=hidden" to SWITCH_AM_CXXFLAGS
  adding "-DSWITCH_API_VISIBILITY=1" to SWITCH_AM_CXXFLAGS
  adding "-DCJSON_API_VISIBILITY=1" to SWITCH_AM_CXXFLAGS
  adding "-DHAVE_VISIBILITY=1" to SWITCH_AM_CXXFLAGS
checking for lua5.2... checking for lua-5.2... checking for lua52... checking for lua5.1... checking for lua-5.1... checking for lua... yes
checking LUA_CFLAGS... -I/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/include 
checking LUA_LIBS... -L/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/lib -llua -lm 
checking for alcLoopbackOpenDeviceSOFT in -lopenal... no
checking for jack... checking for snd_pcm_open in -lasound... yes
checking for uuid >= 1.41.2... yes
checking LIBUUID_CFLAGS... -I/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/include/uuid 
checking LIBUUID_LIBS... -L/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/lib -luuid 
checking for pg_config... /home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/bin/pg_config
checking for pkg-config... /home/mangix/devstuff/openwrt/staging_dir/host/bin/pkg-config
checking for PostgreSQL libraries via pkg_config... -L/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/lib 
checking for PQgetvalue in -lpq... yes
checking for libmariadb >= 3.0.9... yes
checking MARIADB_CFLAGS... -I/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/include/mysql/ 
checking MARIADB_LIBS... -L/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/lib/ -lmariadb 
checking for spandsp3 >= 3.0... yes
checking SPANDSP_CFLAGS... -I/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/lib/spandsp3/include 
checking SPANDSP_LIBS... -L/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/lib/spandsp3/lib -l:libspandsp.so.3 
checking for sofia-sip-ua >= 1.13.6... yes
checking SOFIA_SIP_CFLAGS... -I/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/include/sofia-sip-1.13 
checking SOFIA_SIP_LIBS... -L/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/lib -lsofia-sip-ua 
  setting PLATFORM_CORE_LIBS to "-ldl -lcrypt -lrt"
checking for inflateReset in -lz... yes
  adding "-lz" to PLATFORM_CORE_LIBS
checking for libmpg123 >= 1.16.0... yes
checking MPG123_CFLAGS... -I/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/include 
checking MPG123_LIBS... -L/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/lib -lmpg123 
checking for opencore-amrnb >= 0.1.0... no
checking for opencore-amrwb >= 0.1.0 vo-amrwbenc >= 0.1.0... no
checking for stirshaken... no
checking for apr_pool_mutex_set in -lapr-1... no
checking for apr_queue_pop_timeout in -laprutil-1... no
checking for jpeg_std_error in -ljpeg... yes
checking for jbg_enc_out in -ljbig... no
checking for lzma_code in -llzma... yes
checking for res_init in -lresolv... yes
  adding "-lresolv" to SWITCH_AM_LDFLAGS
checking for dirent.h that defines DIR... yes
checking for library containing opendir... none required
checking for ANSI C header files... (cached) yes
checking for sys/types.h... (cached) yes
checking sys/resource.h usability... yes
checking sys/resource.h presence... yes
checking for sys/resource.h... yes
checking sched.h usability... yes
checking sched.h presence... yes
checking for sched.h... yes
checking wchar.h usability... yes
checking wchar.h presence... yes
checking for wchar.h... yes
checking sys/filio.h usability... no
checking sys/filio.h presence... no
checking for sys/filio.h... no
checking sys/ioctl.h usability... yes
checking sys/ioctl.h presence... yes
checking for sys/ioctl.h... yes
checking sys/prctl.h usability... yes
checking sys/prctl.h presence... yes
checking for sys/prctl.h... yes
checking sys/select.h usability... yes
checking sys/select.h presence... yes
checking for sys/select.h... yes
checking netdb.h usability... yes
checking netdb.h presence... yes
checking for netdb.h... yes
checking execinfo.h usability... no
checking execinfo.h presence... no
checking for execinfo.h... no
checking sys/time.h usability... yes
checking sys/time.h presence... yes
checking for sys/time.h... yes
checking for an ANSI C-conforming const... yes
checking for inline... inline
checking for size_t... yes
checking whether time.h and sys/time.h may both be included... yes
checking whether struct tm is in sys/time.h or time.h... time.h
checking whether ccache_cc needs -traditional... no
checking for stdlib.h... (cached) yes
checking for GNU libc compatible malloc... (cached) yes
checking return type of signal handlers... void
checking for strftime... (cached) yes
checking for gethostname... yes
checking for vasprintf... yes
checking for mmap... yes
checking for mlock... yes
checking for mlockall... yes
checking for usleep... yes
checking for getifaddrs... yes
checking for timerfd_create... yes
checking for getdtablesize... yes
checking for posix_openpt... yes
checking for poll... yes
checking for sched_setscheduler... yes
checking for setpriority... yes
checking for setrlimit... yes
checking for setgroups... yes
checking for initgroups... yes
checking for getrusage... yes
checking for wcsncmp... yes
checking for setgroups... (cached) yes
checking for asprintf... yes
checking for setenv... yes
checking for pselect... yes
checking for gettimeofday... (cached) yes
checking for localtime_r... yes
checking for gmtime_r... yes
checking for strcasecmp... yes
checking for stricmp... no
checking for _stricmp... no
checking whether strerror_r is declared... yes
checking for strerror_r... yes
checking whether strerror_r returns char *... no
checking for sched_setaffinity... yes
checking for sched_getaffinity... yes
checking whether the CPU_SET and CPU_ZERO macros are defined... yes
checking for clock_gettime in -lrt... yes
checking for clock_getres in -lrt... yes
checking for clock_nanosleep in -lrt... yes
checking for pthread_setschedparam in -lpthread... yes
checking for socket... yes
checking for /dev/ptmx... (cached) yes
checking for openpty in -lutil... yes
checking for struct tm.tm_gmtoff... yes
checking for struct tm.tm_zone... yes
checking whether RLIMIT_MEMLOCK is declared... yes
checking whether SCHED_RR is declared... yes
checking whether SCHED_FIFO is declared... yes
checking whether to use mlockall... yes
checking for setenv... (cached) yes
checking for strtoll... yes
checking for strtoull... yes
checking for strtoq... no
checking for strtouq... no
checking for __strtoll... no
checking for __strtoull... no
checking whether va_list is an array... no
checking whether compiler has __attribute__... yes
checking whether compiler supports -Wdeclaration-after-statement... yes
yes
  setting SWITCH_ANSI_CFLAGS to "-Wdeclaration-after-statement"
checking whether byte ordering is bigendian... (cached) no
checking size of char... (cached) 1
checking size of int... (cached) 4
checking size of long... (cached) 8
checking size of short... (cached) 2
checking size of long long... (cached) 8
checking for size_t... (cached) yes
checking for ssize_t... yes
checking size of void*... 8
checking size of ssize_t... (cached) 8
checking size of size_t... (cached) 8
checking for gunzip... /usr/bin/gunzip
checking for bzip2... /home/mangix/devstuff/openwrt/staging_dir/hostpkg/bin/bzip2
checking for xz... /home/mangix/devstuff/openwrt/staging_dir/host/bin/xz
checking for gtar... /usr/bin/gtar
checking for wget... /home/mangix/devstuff/openwrt/staging_dir/host/bin/wget
checking for curl... /usr/bin/curl
checking for gumbo >= 0.10.1... no
checking for libfvad >= 1.0... no
checking for libtpl >= 1.5... no
checking for sqlite3 >= 3.6.20... yes
checking SQLITE_CFLAGS... -I/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/include 
checking SQLITE_LIBS... -L/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/lib -lsqlite3 
checking for libcurl >= 7.19... yes
checking CURL_CFLAGS... -I/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/include 
checking CURL_LIBS... -L/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/lib -lcurl 
checking for libpcre >= 7.8... yes
checking PCRE_CFLAGS... -I/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/include 
checking PCRE_LIBS... -L/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/lib -lpcre 
checking for speex >= 1.2rc1 speexdsp >= 1.2rc1... yes
checking SPEEX_CFLAGS... -I/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/include 
checking SPEEX_LIBS... -L/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/lib -lspeex -lspeexdsp 
checking for yaml-0.1 >= 0.1.4... yes
checking YAML_CFLAGS... -I/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/include 
checking YAML_LIBS... -L/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/lib -lyaml 
checking for portaudio-2.0 >= 19... yes
checking PORTAUDIO_CFLAGS... -I/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/include 
checking PORTAUDIO_LIBS... -L/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/lib -lportaudio -lasound -lm -lpthread 
checking for libldns-fs >= 1.6.6... checking for libldns >= 1.6.6... yes
checking LDNS_CFLAGS... -I/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/include 
checking LDNS_LIBS... -L/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/lib -lldns 
checking for sndfile >= 1.0.20... yes
checking SNDFILE_CFLAGS... -I/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/include 
checking SNDFILE_LIBS... -L/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/lib -lsndfile 
checking for libmpg123 >= 1.16.0... yes
checking MPG123_CFLAGS... -I/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/include 
checking MPG123_LIBS... -L/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/lib -lmpg123 
checking for shout >= 2.2.2... yes
checking SHOUT_CFLAGS... -I/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/include -pthread 
checking SHOUT_LIBS... -L/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/lib -lshout 
checking for lame_init in -lmp3lame... yes
checking lame/lame.h usability... yes
checking lame/lame.h presence... yes
checking for lame/lame.h... yes
checking for libavcodec >= 53.35.0... no
checking for x264 >= 0.142.2431... yes
checking X264_CFLAGS... -I/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/include -DX264_API_IMPORTS 
checking X264_LIBS... -L/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/lib -lx264 
checking for libavformat >= 53.21.1... no
checking for libavutil >= 54.3.0... no
checking for libavresample >= 2.1.0... no
checking for libswresample >= 2.1.0... no
checking for libswscale >= 3.0.0... no
checking for libvlc >= 2.1.0... no
checking for opencv >= 2.4.5... checking for opencv4 >= 2.4.5... no
checking for opusfile >= 0.5... yes
checking OPUSFILE_DECODE_CFLAGS... -I/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/include/opus -I/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/include 
checking OPUSFILE_DECODE_LIBS... -L/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/lib -lopusfile 
checking for libopusenc >= 0.1... yes
checking OPUSFILE_ENCODE_CFLAGS... -I/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/include/opus 
checking OPUSFILE_ENCODE_LIBS... -L/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/lib -lopusenc 
checking for ImageMagick >= 6.0.0... yes
checking MAGICK_CFLAGS... -I/usr/include/ImageMagick-7 -I/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/include/ImageMagick-7 -DMAGICKCORE_HDRI_ENABLE=0 -DMAGICKCORE_QUANTUM_DEPTH=8 
checking MAGICK_LIBS... -L/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/lib -lMagickCore-7.Q8 
checking for ImageMagick >= 7.0.0... yes
checking MAGICK7_CFLAGS... -I/usr/include/ImageMagick-7 -I/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/include/ImageMagick-7 -DMAGICKCORE_HDRI_ENABLE=0 -DMAGICKCORE_QUANTUM_DEPTH=8 
checking MAGICK7_LIBS... -L/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/lib -lMagickCore-7.Q8 
checking for silk >= 1.0.8... no
checking for broadvoice >= 0.1.0... no
checking for ilbc2 >= 0.0.1... checking for ilbc >= 0.0.1... no
checking for g722_1 >= 0.2.0... no
checking for codec2 >= 0.5... checking for codec2_create in -lcodec2... no
checking for opus >= 1.1... yes
checking OPUS_CFLAGS... -I/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/include/opus 
checking OPUS_LIBS... -L/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/lib -lopus 
checking for soundtouch >= 1.7.0... no
checking for flite >= 2... checking for flite_init in -lflite... no
checking for libmongoc-1.0 >= 1.0.8... no
checking for libmemcached >= 0.31... no
checking for v8-6.1_static >= 6.1.298... checking for v8fs_static >= 6.1.298... checking for v8 >= 6.1.298... no
checking for libks >= 1.1.0... yes
checking KS_CFLAGS... -I/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/include/libks 
checking KS_LIBS... -L/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/lib -lks 
checking for signalwire_client >= 1.0.0... yes
checking SIGNALWIRE_CLIENT_CFLAGS... -I/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/include 
checking SIGNALWIRE_CLIENT_LIBS... -L/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/lib -lsignalwire_client 
checking for librabbitmq >= 0.5.2... no
checking for libh2o-evloop >= 0.11.0... yes
checking H2O_CFLAGS... -I/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/include 
checking H2O_LIBS... -L/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/lib -lh2o-evloop 
checking for libbrotlienc >= 0.1.0... no
checking for libbrotlidec >= 0.1.0... no
checking for tap >= 0.1.0... no
checking for libsmpp34 >= 1.10... no
checking for hiredis >= 0.10.0... yes
checking HIREDIS_CFLAGS... -I/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/include/hiredis -D_FILE_OFFSET_BITS=64 
checking HIREDIS_LIBS... -L/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/lib -lhiredis 
checking for libedit >= 2.11... yes
checking LIBEDIT_CFLAGS... -I/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/include -I/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/include/editline 
checking LIBEDIT_LIBS... -L/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/lib -ledit 
checking for ldap_search in -lldap... yes
checking for ber_pvt_opt_on in -llber... yes
checking whether EL_PROMPT_ESC is declared... yes
checking whether EL_REFRESH is declared... yes
checking for el_wset... yes
checking for openssl... yes
checking openssl_CFLAGS... -I/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/include 
checking openssl_LIBS... -L/home/mangix/devstuff/openwrt/staging_dir/target-mips64el_mips64r2_64_musl/usr/lib -lssl -lcrypto 
  adding "-DHAVE_OPENSSL" to SWITCH_AM_CFLAGS
checking for SSL_CTX_set_tlsext_use_srtp in -lssl... yes
checking for DTLSv1_method in -lssl... (cached) yes
checking for DTLSv1_2_method in -lssl... yes
checking for JAVA installation at ... 
configure: cannot find the java directory, assuming it is specified in CFLAGS
checking if JAVA package is complete... no
checking for perl... (cached) false
checking for php... (cached) false
checking for php-config... (cached) false
configure: WARNING: python support disabled, building mod_python will fail!
checking for python3... /home/mangix/devstuff/openwrt/staging_dir/host/bin/python3
checking python3 version... 3.10.0
checking for python3 distutils... no
configure: error: Could not find or use python3 distutils module: <string>:1: DeprecationWarning: The distutils package is deprecated and slated for removal in Python 3.12. Use setuptools or check PEP 632 for potential alternatives
make[2]: *** [Makefile:1122: /home/mangix/devstuff/openwrt/build_dir/target-mips64el_mips64r2_64_musl/freeswitch-1.10.7.-release/.configured_2997c02f277ca094be1d47bf0a1b36a3] Error 1
make[2]: Leaving directory '/home/mangix/devstuff/openwrt/feeds/telephony/net/freeswitch'
time: package/feeds/telephony/freeswitch/compile#37.80#4.38#0.00
    ERROR: package/feeds/telephony/freeswitch failed to build.
make[1]: *** [package/Makefile:116: package/feeds/telephony/freeswitch/compile] Error 1
make[1]: Leaving directory '/home/mangix/devstuff/openwrt'
make: *** [/home/mangix/devstuff/openwrt/include/toplevel.mk:230: package/freeswitch/compile] Error 2

```

--- a/net/freeswitch/Makefile
+++ b/net/freeswitch/Makefile
@@ -234,7 +234,7 @@ include $(INCLUDE_DIR)/nls.mk
 
 FS_PERL_FEED:=$(TOPDIR)/feeds/packages/lang/perl
 
-include $(TOPDIR)/feeds/packages/lang/python/python3-host.mk
+include $(TOPDIR)/feeds/packages/lang/python/python3-version.mk
 include $(FS_PERL_FEED)/perlver.mk
 
 PERL_SITELIB:=/usr/lib/perl$(PERL_MAJOR)/$(PERL_VERSION2)

--- a/net/freeswitch/patches/460-remove-python-configure-checks.patch
+++ b/net/freeswitch/patches/460-remove-python-configure-checks.patch
@@ -1,6 +1,25 @@
 --- a/configure.ac
 +++ b/configure.ac
-@@ -1865,24 +1865,12 @@ then
+@@ -1853,36 +1853,24 @@ then
+ 		fi
+ 		AC_MSG_RESULT([$PYTHON3_VER])
+ 
+-		AC_MSG_CHECKING([for python3 distutils])
+-		python3_result="`$PYTHON3 -c 'import distutils;' 2>&1`"
+-		if test -z "$python3_result" ; then
++		#AC_MSG_CHECKING([for python3 distutils])
++		#python3_result="`$PYTHON3 -c 'import distutils;' 2>&1`"
++		#if test -z "$python3_result" ; then
+ 			python3_has_distutils="yes"
+-		else
+-			python3_has_distutils="no"
+-		fi
+-		AC_MSG_RESULT([$python3_has_distutils])
++		#else
++		#	python3_has_distutils="no"
++		#fi
++		#AC_MSG_RESULT([$python3_has_distutils])
+ 
  		if test "$python3_has_distutils" != "no" ; then
  			AC_MSG_CHECKING([location of python3 site-packages])
  

--- a/net/freeswitch/patches/480-fix-libyuv-dependency.patch
+++ b/net/freeswitch/patches/480-fix-libyuv-dependency.patch
@@ -1,55 +1,87 @@
---- a/src/include/switch_core_video.h
-+++ b/src/include/switch_core_video.h
-@@ -394,6 +394,7 @@ SWITCH_DECLARE(switch_image_t *) switch_
- SWITCH_DECLARE(switch_status_t) switch_img_write_png(switch_image_t *img, char *file_name);
- SWITCH_DECLARE(switch_status_t) switch_png_open(switch_png_t **pngP, const char *file_name);
- SWITCH_DECLARE(void) switch_png_free(switch_png_t **pngP);
-+#ifdef SWITCH_HAVE_YUV
- SWITCH_DECLARE(switch_status_t) switch_img_data_url_png(switch_image_t *img, char **urlP);
- SWITCH_DECLARE(switch_status_t) switch_img_data_url(switch_image_t *img, char **urlP, const char *type, int quality);
- 
-@@ -405,6 +406,7 @@ SWITCH_DECLARE(switch_image_t *) switch_
- * \param[in]    quality   Only used in jpg, 1 ~ 100
- */
- SWITCH_DECLARE(switch_status_t) switch_img_write_to_file(switch_image_t *img, const char* file_name, int quality);
-+#endif
- 
- /*!\brief put a small img over a big IMG at position x,y, with alpha transparency
- *
+commit a2ce46c6fde38d6ac54a8a2ee1a5b391e2ed2071
+Author: Sebastian Kemper <sebastian_ml@gmx.net>
+Date:   Mon Nov 1 09:59:09 2021 +0100
+
+    [core] fix "--disable-libyuv"
+    
+    Recent changes made it impossible to compile freeswitch without libyuv
+    support.
+    
+    src/switch_core_video.c: In function 'switch_img_read_from_file':
+    src/switch_core_video.c:3139:4: error: implicit declaration of function 'RAWToI420' [-Werror=implicit-function-declaration]
+    RAWToI420(data, width * 3,
+    ^
+    src/switch_core_video.c:3148:4: error: implicit declaration of function 'ABGRToARGB' [-Werror=implicit-function-declaration]
+    ABGRToARGB(data, width * 4, img->planes[SWITCH_PLANE_PACKED], img->stride[SWITCH_PLANE_PACKED], width, height);
+    ^
+    
+    Fix this my adding/moving the checks for "SWITCH_HAVE_YUV".
+    
+    Signed-off-by: Sebastian Kemper <sebastian_ml@gmx.net>
+
 --- a/src/switch_core_video.c
 +++ b/src/switch_core_video.c
-@@ -48,13 +48,13 @@
- #include <gd.h>
- #endif
+@@ -3116,6 +3116,7 @@ SWITCH_DECLARE(switch_status_t) switch_i
  
-+#ifdef SWITCH_HAVE_YUV
- #define STB_IMAGE_IMPLEMENTATION
- #include "../libs/stb/stb_image.h"
- 
- #define STB_IMAGE_WRITE_IMPLEMENTATION
- #include "../libs/stb/stb_image_write.h"
- 
--#ifdef SWITCH_HAVE_YUV
- static inline void switch_img_get_yuv_pixel(switch_image_t *img, switch_yuv_color_t *yuv, int x, int y);
- #endif
- 
-@@ -3114,6 +3114,7 @@ SWITCH_DECLARE(switch_status_t) switch_i
- 
- #endif
- 
-+#ifdef SWITCH_HAVE_YUV
  SWITCH_DECLARE(switch_image_t *) switch_img_read_from_file(const char* file_name, switch_img_fmt_t img_fmt)
  {
++#ifdef SWITCH_HAVE_YUV
  	int width = 0, height = 0, channels = 0;
-@@ -3301,7 +3302,7 @@ SWITCH_DECLARE(switch_status_t) switch_i
+ 	int comp = STBI_rgb;
+ 	unsigned char *data = NULL;
+@@ -3155,12 +3156,16 @@ SWITCH_DECLARE(switch_image_t *) switch_
+ 	} else if (data) {
+ 		stbi_image_free(data);
+ 	}
++#endif
+ 
+ 	return NULL;
+ }
+ 
+ SWITCH_DECLARE(switch_status_t) switch_img_write_to_file(switch_image_t *img, const char* file_name, int quality)
+ {
++#ifndef SWITCH_HAVE_YUV
++	return SWITCH_STATUS_FALSE;
++#else
+ 	int comp = STBI_rgb;
+ 	unsigned char *data = NULL;
+ 	const char *ext = strrchr(file_name, '.');
+@@ -3217,6 +3222,7 @@ SWITCH_DECLARE(switch_status_t) switch_i
+ 	free(data);
  
  	return ret ? SWITCH_STATUS_SUCCESS : SWITCH_STATUS_FALSE;
++#endif
  }
--
-+#endif /* SWITCH_HAVE_YUV */
  
- SWITCH_DECLARE(switch_status_t) switch_img_letterbox(switch_image_t *img, switch_image_t **imgP, int width, int height, const char *color)
+ typedef struct data_url_context_s {
+@@ -3224,14 +3230,19 @@ typedef struct data_url_context_s {
+ 	char **urlP;
+ } data_url_context_t;
+ 
++#ifdef SWITCH_HAVE_YUV
+ static void data_url_write_func(void *context, void *data, int size)
  {
+ 	switch_buffer_t *buffer = (switch_buffer_t *)context;
+ 	switch_buffer_write(buffer, data, size);
+ }
++#endif
+ 
+ SWITCH_DECLARE(switch_status_t) switch_img_data_url(switch_image_t *img, char **urlP, const char *type, int quality)
+ {
++#ifndef SWITCH_HAVE_YUV
++	return SWITCH_STATUS_FALSE;
++#else
+ 	int comp = STBI_rgb;
+ 	unsigned char *data = NULL;
+ 	int stride_in_bytes = 0;
+@@ -3300,6 +3311,7 @@ SWITCH_DECLARE(switch_status_t) switch_i
+ 	switch_buffer_destroy(&buffer);
+ 
+ 	return ret ? SWITCH_STATUS_SUCCESS : SWITCH_STATUS_FALSE;
++#endif /* SWITCH_HAVE_YUV */
+ }
+ 
+ 
 --- a/tests/unit/switch_core_video.c
 +++ b/tests/unit/switch_core_video.c
 @@ -48,6 +48,7 @@ FST_CORE_BEGIN("./conf")

